build.mill

```diff
@@ -188,6 +188,9 @@ object Deps {
   val sonatypeCentralClient = ivy"com.lumidion::sonatype-central-client-requests:0.3.0"
   val kotlinCompiler = ivy"org.jetbrains.kotlin:kotlin-compiler:1.9.24"
   val koverVersion = "0.8.3"
+  val ktfmtVersion = "0.52"
+  val detektVersion = "1.23.7"
+  val dokkaVersion = "1.9.20"
```

Comment:
I think some or all of them are runtime versions, not needed to build the modules. So, they should go into the dedicated `object RuntimeDeps` below. Make sure to add them to `all` too, so we can check for updates automatically.